votor/src/voting_service.rs

```diff
@@ -0,0 +1,366 @@
+use {
+    crate::{
+        staked_validators_cache::StakedValidatorsCache, vote_history_storage::VoteHistoryStorage,
+        voting_utils::BLSOp,
```

Comment:
maybe just move this into here since we're in the votor crate now

Reply:
Done.